Componets/Avatar/avatar.css

```diff
@@ -0,0 +1,30 @@
+.avatar-container{
+    margin: 2rem;;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+}
+.avatar{
+    border-radius: 50%;
+    border: 1x solid var(--secondary-color)
```

Comment:
You mistyped "1x" instead of "1px", try changing this.

Reply:
Typing mistake...corrected. Thanks for feedback.

Reply:
`.avatar{
    border-radius: 50%;
    border: 1x solid var(--secondary-color)
}`